core/templates/services/state-top-answers-stats.service.spec.ts

```diff
@@ -279,7 +279,7 @@ describe('StateTopAnswersStatsService', function() {
     it('throws when fetching stats about non-existent states', function() {
       expect(function() {
         StateTopAnswersStatsService.getStateStats('Me Llamo');
-      }).toThrow();
+      }).toThrowError( 'Me Llamo does not exist.');
```

Comment:
Only see in this file, but please make sure there aren't any extra spaces being added like this example.

Reply:
Nice catch! Sure, I'll double-check this.

Reply:
Done.